flake.nix

```diff
@@ -25,8 +31,6 @@
             };
           };
 
-          overlays = ./overlays;
```

Comment:
Just to be sure: This nested overlays field is not part of the standard flake specification, right?

Reply:
Yeah, as far as I can see it should be toplevel in the output schema, and not branched with `flake-utils.lib.eachSystem`.

Reply:
I have no idea, but I thought the overlays didn't need to be generated for each system.